recipe/install_openssl.sh

```diff
@@ -11,6 +11,11 @@ if [[ ${HOST} =~ .*linux.* ]]; then
   fi
 fi
 
+# Make sure ${PREFIX}/ssl/certs directory exists
+# Otherwise SSL_ERROR_SYSCALL is returned instead of SSL_ERROR_SLL
+mkdir -p ${PREFIX}/ssl/certs
+touch ${PREFIX}/ssl/certs/.keep
```

Comment:
Could we quote these?

```suggestion
# Make sure ${PREFIX}/ssl/certs directory exists
# Otherwise SSL_ERROR_SYSCALL is returned instead of SSL_ERROR_SLL
mkdir -p "${PREFIX}/ssl/certs"
touch "${PREFIX}/ssl/certs/.keep"
```